### PR TITLE
Allow JSON to handle comments

### PR DESCRIPTION
**Pull request by yyyc514**

### CODE REVIEW

**Comment by yyyc514** [comment]
Github calls this "jsonc" in one place but I'm not sure if that is actually a real "thing" or not.  If we want I can add it as an alias, but I didn't so far.

File: test/markup/index.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ function testLanguage(language) {
 
           actual.trim().should.equal(expected.trim());
           done();
-        });
+        }).catch(function(err) { return done(err) });
```
Comment (egor-rogov) [inline test/markup/index.js]: Wonderful (:

**Comment by yyyc514** [comment]
Awesomeness.

**Comment by egor-rogov** [comment]
Thanks, merged!

**Pull request merged**